### PR TITLE
Add template of email

### PR DESCRIPTION
**Pull request by HumbertoMLL**
Add template of data access

### CODE REVIEW

**Comment by ealexisaraujo** [comment]
@hyfi06 I checked email template, I downloaded template and execute example. Check screen:

And If it's correct we can merge.

Screen Shot 2020-05-25 at 9 49 15

File: utils/templates/email/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      text-decoration: none;
+    }
+  </style>
+  <body>
+    <main>
+      <img
+        src="https://i.imgur.com/JR05wxd.png"
+        alt="Logo of HALAH Laboratories"
+      />
+      <h1>Welcome Héctor</h1>
```
Comment (hyfi06) [inline utils/templates/email/index.html:53]: `Welcome firstName`

File: utils/templates/email/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      <p>
+        HALAH Laboratories <br />
+        thanks you for your preference
+      </p>
+      <div>
+        <p>This are your access data</p>
+        <p>Username: <span>hector.olvera.7287</span></p>
```
Comment (hyfi06) [inline utils/templates/email/index.html:60]: `<p>Username: <span>username</span></p>`

File: utils/templates/email/index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        <p>Password: <span>123456789123</span></p>
```
Comment (hyfi06) [inline utils/templates/email/index.html:61]: `<p>Password: <span>password</span></p>`

File: utils/templates/email/index.html (hunk 1 above)
Comment (ealexisaraujo) [reply utils/templates/email/index.html]: Done!

File: utils/templates/email/index.html (hunk 2 above)
Comment (ealexisaraujo) [reply utils/templates/email/index.html]: Done!

File: utils/templates/email/index.html (hunk 3 above)
Comment (ealexisaraujo) [reply utils/templates/email/index.html]: Done!

**Pull request merged**